src/renderer/components/ConnectivityToast.tsx

```diff
@@ -1,21 +1,16 @@
-import React, {
-  useEffect,
-  useState,
-  useCallback,
-  useMemo,
-  useContext,
-} from 'react'
-
-import { getLogger } from '../../shared/logger'
-import { ScreenContext, useTranslationFunction } from '../contexts'
-import { useKeyBindingAction, KeybindAction } from '../keybindings'
+import React, { useEffect, useState, useCallback, useMemo } from 'react'
 import { C } from '@deltachat/jsonrpc-client'
-
 import { debounce } from 'debounce'
+
+import { getLogger } from '../../shared/logger'
+import { KeybindAction } from '../keybindings'
 import { debounceWithInit } from './chat/ChatListHelpers'
 import SettingsConnectivityDialog from './dialogs/Settings-Connectivity'
 import { BackendRemote, onDCEvent } from '../backend-com'
 import { selectedAccountId } from '../ScreenController'
+import { useDialog } from '../hooks/useDialog'
+import { useTranslationFunction } from '../hooks/useTranslationFunction'
+import { useKeyBindingAction } from '../hooks/useKeyBindingAction'
```

Comment:
Each hook became its own file, which hopefully makes it easier now to find the code faster when one wants to look at it / change it